riocli/parameter/utils.py

```diff
@@ -91,3 +95,8 @@ def phase3(self) -> None:
                                   self.common_files,
                                   shallow=False)
         self.same_files, self.diff_files, self.funny_files = f_comp
+
+
+def is_valid_tree_name(name: str) -> bool:
+    """Validates a config tree name"""
+    return bool(re.match(r'^[a-zA-Z0-9_-]{3,50}$', name))
```

Comment:
I've updated this regex in my flaptter fixes. Can you update this validation as well to match that.